### DISCUSSION TITLE
Collection highlighter

### DISCUSSION THREAD

**Discussion by tehervonat** [fedorovvl/tso_client#47] [Q&A] [answered]
Hello! Can i use collection highlighter with this client?

**Comment by BasilMas** [comment]
UbiCollect only, no Pinky

**Comment by tehervonat** [comment, marked as answer]
And how? Can you help me?

**Reply by BasilMas** [reply]
download it from https://sirris.tsomaps.com/UbiCollect2_2.zip

Unzip, start and enjoy)
Don't forget to create the shortcut of the program in startup menu for starting automatically after Windows restart
 
See in details here
https://settlersonlinewiki.eu/en/playing-on-unity-by-the-client/

**Reply by tehervonat** [reply]
Thank you! It is working now.
